File: src/main/java/com/rimlang/rim/syntax/ExpressionSyntaxNode.java
package com.rimlang.rim.syntax;

import com.rimlang.rim.lexer.Token;

import java.util.List;

public class ExpressionSyntaxNode extends SyntaxNode {
    private final List<Token> code;

    public ExpressionSyntaxNode(List<Token> code) {
        this.code = code;
    }

    public List<Token> getCode() {
        return code;
    }

}

File: src/main/kotlin/com/mcpy/lang/syntax/node/expression/ExpressionSyntaxNode.kt
package com.mcpy.lang.syntax.node.expression

import com.mcpy.lang.lexer.token.Token
import com.mcpy.lang.syntax.node.SyntaxNode

data class ExpressionSyntaxNode(val code: List<Token>, override val firstToken: Token) : SyntaxNode(firstToken)
